RFC/src/RFC-0120_Consensus.md

```diff
@@ -140,18 +144,44 @@ The timestamp MUST confirm to the following:
 
 #### Output_mr
 
-This is the merkle root of the outputs. This is calculated in the following way: Hash (txo MMR root  || roaring bitmap hash of UTXO indices).
+This is the merkle root of the outputs. It MUST be calculated in the following way: Hash (`txo MMR root`  || 
+Hash(`roaring bitmap`)). The output_mr field is used to represent the entire UTXO set in the blockchain; this is a 
+proof that every UTXO exists and that a TXO is either spent or unspent. The `txo MMR root` MUST be the merkle 
+mountain range (MMR) root of all transactional outputs in existence. The `roaring bitmap hash` MUST be a roaring bitmap 
+of every spent transactional output (aka every input in a block). The index used in the roaring bitmap represents the 
+leaf index of the UTXO in the output MMR. 
 
 The output_mr MUST confirm to the following:
 
 * Represented as an array of unsigned 8-bit integers (bytes) in little-endian format.
 * The hashing function used MUST be blake2b with a 256 bit digest.
 
-#### Range_proof_mr
+#### Output_mmr_size
 
-This is the merkle root of the range proofs.
+This is the total size of the leaves in the output merkle mountain range.
 
-The range_proof_mr MUST confirm to the following:
+The Output_mmr_size MUST confirm to the following:
```

Comment:
Should this not be conform? This error also appears in other places in this document 

```suggestion
The Output_mmr_size MUST conform to the following:
```